src/resources/Enums.ts

```diff
@@ -1200,6 +1200,8 @@ export enum ApiKeyStatusFilter {
     SOON_TO_BE_EXPIRED = 'SOON_TO_BE_EXPIRED',
     /** Keys that are active but have been flagged as exposed */
     ACTIVE_AND_EXPOSED = 'ACTIVE_AND_EXPOSED',
+    /** All key that are exposed */
+    ALL_EXPOSED = 'EXPOSED',
```

Comment:
I think we should use the same name as the backend
```suggestion
    /** Keys that have been exposed */
    EXPOSED = 'EXPOSED',
```

Reply:
Sorry for that 😭 . It's `ALL_EXPOSED` in the backend